recipe/0002-Allow-all-ruamel.yaml-0.16-versions-in-setup.py.patch

```diff
@@ -0,0 +1,11 @@
+--- setup.py
++++ setup.py
+@@ -29,7 +29,7 @@
+ install_requires = [
+     'setuptools',
+     'requests >= 1.0',
+-    'ruamel.yaml >= 0.12.4, <= 0.16',
++    'ruamel.yaml >= 0.12.4, < 0.17',
```

Comment:
@conda-forge/schema-salad, is this okay to do, or is the requirement intentionally restricted with an upper bound of `0.16.0`?